datasets/streaming/error_aggregates/reference.md

```diff
@@ -8,16 +8,54 @@
 
 # Data Reference
 
-## Sampling
+## Example Queries
+
+Getting a large number of different crash measures across many platforms and channels
+([view on redash](https://sql.telemetry.mozilla.org/queries/4769/source)):
+
+```sql
+SELECT window_start,
+       build_id,
+       channel,
+       os_name,
+       version,
+       sum(usage_hours) AS usage_hours,
+       sum(main_crashes) AS main,
+       sum(content_crashes) AS content,
+       sum(gpu_crashes) AS gpu,
+       sum(plugin_crashes) AS plugin,
+       sum(gmplugin_crashes) AS gmplugin
+FROM telemetry.error_aggregates
+  WHERE application = 'Firefox' 
+  AND (os_name = 'Darwin' or os_name = 'Linux' or os_name = 'Windows_NT')
+  AND (channel = 'beta' or channel = 'release' or channel = 'nightly' or channel = 'esr')
+  AND build_id > '201801'
+  AND window_start > current_timestamp - (1 * interval '24' hour)
+GROUP BY window_start, channel, build_id, version, os_name
+```
+
+
+Get the number of main_crashes on Windows over a small interval
+([view on redash](https://sql.telemetry.mozilla.org/queries/51677)):
+
+```sql
+SELECT window_start as time, sum(main_crashes) AS main_crashes
+FROM telemetry.error_aggregates_v1
+  WHERE application = 'Firefox'
+  AND os_name = 'Windows_NT'
+  AND channel = 'release'
+  AND version = '58.0.2'
+  AND window_start > timestamp '2018-02-21'
+  AND window_end < timestamp '2018-02-22'
+GROUP BY window_start
+```
 
-### Invalid Pings
+## Sampling
 
-We ignore invalid pings in our processing. Invalid pings are defined as those that:
+### Data sources
 
-* The submission dates or activity dates are invalid or missing.
-* The build ID is malformed.
-* The docType field is missing or unknown.
-* The ping does not include a positive count of usage hours.
+The aggregates in this data source are derived from main and crash [pings](../../pings.md).
+Crash pings are only used to count/gather main crash events, all other errors are gathered from main ones.
```

Comment:
s/main ones/main pings/

Reply:
I was trying not to overuse the word "pings", but it's probably less awkward your way (which was also my original wording). Will change.